net/nginx/files/acme.hotplug

```diff
@@ -0,0 +1 @@
+[ renewed = "$ACTION" ] && /etc/init.d/nginx reload
```

Comment:
As I mentioned in the OP, I think messing with user's config is a mistake, it's dangerous and flaky. Requiring the user to run /usr/bin/acme at least once before specifying ssl in a httpd solves all the problems, and is something reasonable to require IMHO.

Reply:
Right, but there's also a backwards compatibility issue here (see my other comment below). Now, arguably if you're upgrading and have already issued a certificate, your nginx config should already contain the cert path, so as long as that stays stable things should continue to work.

However, we should at least detect if the different `update_*` variables are set in the config, and emit a deprecation warning (maybe combined with a grep for `$domain_dir` in the config to get an indication as to whether things will continue to work?)

Reply:
I see your point. Although with the new design, a httpd is supposed to be automatically updated, and the `update_*` options are simply ignored. I wonder if it's necessary to point this out in a warning log, since nothing should be broken and continue to work?

Reply:
Right, plain nginx will continue to work on an upgrade as long as `$domain_dir` stays the same, which it does. However, haproxy won't because that relies on a different name, and uhttpd won't because that doesn't have a reload hook anymore. Also, nginx-util is no longer called, so anyone using that will break.